setup.py

```diff
@@ -300,7 +300,8 @@ def run(self):
             raise RuntimeError("No build directory. Run `python setup.py build` before trying to run tests.")
         os.environ['GRAMPS_RESOURCES'] = '.'
         all_tests = unittest.TestLoader().discover('.', pattern='*_test.py')
-        unittest.TextTestRunner(verbosity=self.verbose).run(all_tests)
+        unittest.TextTestRunner(
+            verbosity=2 if self.verbose else 0).run(all_tests)
```

Comment:
This causes the tests to be run in verbose mode even if the `-v` option is not specified.